design/move-gh-org.md

```diff
@@ -0,0 +1,89 @@
+# Plan for moving the Velero GitHub repo into the VMware GitHub organization
+
+Currently, the Velero repository sits under the Heptio GitHub organization. With the acquisition of Heptio by VMware, it is due time that this repo moves to one of the VMware GitHub organizations. This document outlines a plan to move this repo to the VMware Tanzu (https://github.com/vmware-tanzu) organization.
+
+## Goals
+
+- List all steps necessary to have this repo fully functional under the new org
+
+## Non Goals
+
+- Highlight any step necessary around setting up the new organization and its members
+
+## Action items
+
+### Todo list
+
+#### Pre move
+
+- [ ] PR: Blog post communicating the move. https://github.com/heptio/velero/issues/1841. Who: TBD.
```

Comment:
We probably also want to communicate this via the mailing list/slack/twitter and any other means too, right? For those, we can just point to this blog post.

Reply:
Yes, the usual.